src/OpenTelemetry/Metrics/ExponentialHistogramBuckets.cs

```diff
@@ -21,14 +21,15 @@ namespace OpenTelemetry.Metrics;
 public sealed class ExponentialHistogramBuckets
 {
     private long[] buckets = Array.Empty<long>();
+    private int size = 0;
```

Comment:
nit: This initialization is probably not needed,
```suggestion
    private int size;
```